config/main.json

```
@@ -1,15 +1,15 @@
 {
   "chartVersions": {
-    "cert-manager": "v0.13.0",
-    "external-dns": "2.19.2",
-    "grafana": "5.0.5",
+    "cert-manager": "v0.14.3",
+    "external-dns": "2.20.4",
+    "grafana": "5.4.1",
     "loki-stack": "0.32.1",
-    "matrixbot": "v1.0.1",
-    "nginx-ingress": "1.33.4",
-    "polkadot": "v0.17.5",
-    "prometheus-node-exporter": "1.9.0",
-    "prometheus-operator": "8.11.1",
-    "substrate-telemetry": "v1.0.2"
+    "matrixbot": "v1.1.1",
+    "nginx-ingress": "1.41.1",
+    "polkadot": "v0.29.0",
```

Let's keep the old version of polkadot charts to prevent surprises (given that this is going to be used in kusama core infra). I'm pretty sure that there are no breaking changes (given the minor version bump) but to be extra sure.
```suggestion
    "polkadot": "v0.17.5",
```

---

should we stick with this decision even for this PR ? https://github.com/w3f/polkadot-deployer/pull/73/files#diff-b14a753d0a9561dd79e258e43e3067c9R9

---

We are not using v0 for anything, and I'm pretty sure that anyone is using it. But i agree that we should be consistent and do the same for all the previous version branches.

---

Proposed here https://github.com/w3f/polkadot-deployer/pull/75